src/main/java/com/minecolonies/coremod/entity/ai/mobs/util/BarbarianSpawnUtils.java

```diff
@@ -0,0 +1,220 @@
+    public static final ResourceLocation ArcherLootTable    = new ResourceLocation(Constants.MOD_ID, "EntityArcherBarbarianDrops");
+    public static final ResourceLocation ChiefLootTable     = new ResourceLocation(Constants.MOD_ID, "EntityChiefBarbarianDrops");
+
+    /**
+     * Barbarian Attack Damage.
+     */
+    public static final double ATTACK_DAMAGE = 2.0D;
+
+    /**
+     * Values used in Spawn() method
+     */
+    private static final float WHOLE_CIRCLE = 360.0F;
+
+    /**
+     * Values used for AI Task's Priorities.
+     */
+    private static final int    PRIORITY_ZERO               = 0;
+    private static final int    PRIORITY_ONE                = 1;
+    private static final int    PRIORITY_TWO                = 2;
+    private static final int    PRIORITY_THREE              = 3;
+    private static final int    PRIORITY_FOUR               = 4;
+    private static final int    PRIORITY_FIVE               = 5;
+    /**
+     * Other various values used for AI Tasks.
+     */
+    private static final double AI_MOVE_SPEED               = 2.0D;
+    private static final float  MAX_WATCH_DISTANCE          = 8.0F;
+    /**
+     * Values used for mob attributes.
+     */
+    private static final double FOLLOW_RANGE                = 35.0D;
+    private static final double MOVEMENT_SPEED              = 0.2D;
+    private static final double ARMOR                       = 2.0D;
+    private static final double BARBARIAN_BASE_HEALTH       = 10;
+    private static final double BARBARIAN_HEALTH_MULTIPLIER = 0.25;
+
+    /**
+     * Private constructor to hide the implicit public one.
+     */
+    private BarbarianSpawnUtils()
+    {
+    }
+
+
+    /**
+     * Set barbarian attributes.
+     *
+     * @param barbarian The barbarian to set the attributes on.
+     * @param colony    The colony that the barbarian is attacking.
+     */
+    public static void setBarbarianAttributes(final AbstractEntityBarbarian barbarian,final Colony colony)
+    {
+        barbarian.getEntityAttribute(SharedMonsterAttributes.FOLLOW_RANGE).setBaseValue(FOLLOW_RANGE);
+        barbarian.getEntityAttribute(SharedMonsterAttributes.MOVEMENT_SPEED).setBaseValue(MOVEMENT_SPEED);
+        barbarian.getEntityAttribute(SharedMonsterAttributes.ATTACK_DAMAGE).setBaseValue(ATTACK_DAMAGE);
+        barbarian.getEntityAttribute(SharedMonsterAttributes.ARMOR).setBaseValue(ARMOR);
+        barbarian.getEntityAttribute(SharedMonsterAttributes.ARMOR).setBaseValue(getHealthBasedOnRaidLevel(colony));
+    }
+
+    /**
+     * Sets the entity's health based on the raidLevel
+     *
+     * @return returns the health in the form of a double
+     */
+    private static double getHealthBasedOnRaidLevel(final Colony colony)
+    {
+        if (colony != null)
+        {
+            final int raidLevel = (int) (MobEventsUtils.getColonyRaidLevel(colony) * BARBARIAN_HEALTH_MULTIPLIER);
+            return BARBARIAN_BASE_HEALTH + (raidLevel - 10);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Assign this magic number 10 to a well-named constant, and use the constant instead. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS109)